test/NuGet.Core.FuncTests/NuGet.XPlat.FuncTest/XPlatConfigTests.cs

```diff
@@ -0,0 +1,109 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.IO;
+using System.Linq;
+using NuGet.CommandLine.XPlat;
+using NuGet.Test.Utility;
+using Xunit;
+
+namespace NuGet.XPlat.FuncTest
+{
+    [Collection("NuGet XPlat Config Test Collection")]
+    public class XPlatConfigTests
+    {
+
+        [Fact]
+        public void ConfigPathsCommand_ListConfigPathsWithArgs_Success()
+        {
+            // Arrange
+            using (var testInfo = new TestInfo("NuGet.Config"))
+            {
+                var args = new[]
+                {
+                    "config",
+                    "paths",
+                    testInfo.WorkingPath
+                };
+                var log = new TestCommandOutputLogger();
+
+                // Act
+                var exitCode = Program.MainInternal(args.ToArray(), log);
+
+                // Assert
+                Assert.Equal(string.Empty, log.ShowErrors());
+                Assert.Equal(0, exitCode);
+                Assert.Contains(Path.Combine(testInfo.WorkingPath.Path, "NuGet.Config"), log.Messages);
+            }
+        }
+
+        [Fact]
+        public void ConfigPathsCommand_ListConfigPathsNonExistingDirectory_Fail()
+        {
+            // Arrange
+            using (var testInfo = new TestInfo("NuGet.Config"))
+            {
+                var args = new[]
+                {
+                    "config",
+                    "paths",
+                    @"C:\Test\NonExistingRepos"
+                };
+                var log = new TestCommandOutputLogger();
+
+                // Act
+                var exitCode = Program.MainInternal(args.ToArray(), log);
+                var expectedError = @"The path 'C:\Test\NonExistingRepos' doesn't exist.";
+
+                // Assert
+                Assert.Contains(expectedError, log.ShowErrors());
+                Assert.Equal(1, exitCode);
+            }
+        }
+
+        internal class TestInfo : IDisposable
```

Comment:
I think this design works in the case where you'll eventually want more test methods which don't need a `TestInfo`, so you can choose whether to create one, and maybe add parameters to it later. 
However, some ideas I had that you could consider:
1. If you're going to have a mix of tests which do and do not need `TestInfo`, maybe best to keep what you have.
1. If you want the exact `TestInfo` in each test (as currently is the case), you could use a constructor/dispose on the test class directly (without a separate class). See https://xunit.net/docs/shared-context.
1. Consider if it makes sense to create 2 (or more) test classes with different contexts (like another variation of your `TestInfo`) for the same source code. It doesn't have to be 1-to-1 for source-to-test class even though a lot of our test projects cram everything into one.
1. FYI, our test architecture varies widely across projects, so don't be hesitant to challenge any patterns we already have.

Reply:
Hi Donnie! I'm going to add more tests to this file but I'm not 100% sure what they'll look like yet, so for now I think it'll be good to keep these tests in this format, and then try implementing the ideas you suggested for future tests.